_releases/0.21.0.md

```diff
@@ -0,0 +1,1362 @@
+  #19008, #19041, #19164, #19201, #19267, #19276, #19321, #19371,
+  #19427, #19730, #19746, #19881, #20294, #20339, #20368 CI improvements (MarcoFalke)
+- #20489, #20506 MSVC CI improvements (sipsorcery)
+
+### Miscellaneous
+- #18713 scripts: Add macho stack canary check to security-check.py (fanquake)
+- #18629 scripts: Add pe .reloc section check to security-check.py (fanquake)
+- #18437 util: `Detect posix_fallocate()` instead of assuming (vasild)
+- #18413 script: Prevent ub when computing abs value for num opcode serialize (pierreN)
+- #18443 lockedpool: avoid sensitive data in core files (FreeBSD) (vasild)
+- #18885 contrib: Move optimize-pngs.py script to the maintainer repo (MarcoFalke)
+- #18317 Serialization improvements step 6 (all except wallet/gui) (sipa)
+- #16127 More thread safety annotation coverage (ajtowns)
+- #19228 Update libsecp256k1 subtree (sipa)
+- #19277 util: Add assert identity function (MarcoFalke)
+- #19491 util: Make assert work with any value (MarcoFalke)
+- #19205 script: `previous_release.sh` rewritten in python (bliotti)
+- #15935 Add <datadir>/settings.json persistent settings storage (ryanofsky)
+- #19439 script: Linter to check commit message formatting (Ghorbanian)
+- #19654 lint: Improve commit message linter in travis (fjahr)
+- #15382 util: Add runcommandparsejson (Sjors)
+- #19614 util: Use `have_fdatasync` to determine fdatasync() use (fanquake)
+- #19813 util, ci: Hard code previous release tarball checksums (hebasto)
+- #19841 Implement Keccak and `SHA3_256` (sipa)
+- #19643 Add -netinfo peer connections dashboard (jonatack)
+- #15367 feature: Added ability for users to add a startup command (benthecarman)
+- #19984 log: Remove static log message "Initializing chainstate Chainstate [ibd] @ height -1 (null)" (practicalswift)
+- #20092 util: Do not use gargs global in argsmanager member functions (hebasto)
+- #20168 contrib: Fix `gen_key_io_test_vectors.py` imports (MarcoFalke)
+- #19624 Warn on unknown `rw_settings` (MarcoFalke)
+- #20257 Update secp256k1 subtree to latest master (sipa)
+- #20346 script: Modify security-check.py to use "==" instead of "is" for literal comparison (tylerchambers)
+- #18881 Prevent UB in DeleteLock() function (hebasto)
+- #19180, #19189, #19190, #19220, #19399 Replace RecursiveMutex with Mutex (hebasto)
+- #19347 Make `cs_inventory` nonrecursive (jnewbery)
+- #19773 Avoid recursive lock in IsTrusted (promag)
+- #18790 Improve thread naming (hebasto)
+- #20140 Restore compatibility with old CSubNet serialization (sipa)
+- #17775 DecodeHexTx: Try case where txn has inputs first (instagibbs)
+
+### Documentation
+- #18502 Update docs for getbalance (default minconf should be 0) (uzyn)
+- #18632 Fix macos comments in release-notes (MarcoFalke)
+- #18645 Update thread information in developer docs (jnewbery)
+- #18709 Note why we can't use `thread_local` with glibc back compat (fanquake)
+- #18410 Improve commenting for coins.cpp|h (jnewbery)
```

Comment:
This pipe needs to be escaped or contained in backtics, otherwise it creates a markdown table here.